MOM_input

```diff
@@ -154,48 +89,63 @@ SAVE_INITIAL_CONDS = True
 
-COORD_CONFIG = none
-                                ! "This specifies how layers are to be defined:
-                                ! file - read coordinate information from the file
-                                ! specified by (COORD_FILE).
-                                ! linear - linear based on interfaces not layesrs.
-                                ! ts_ref - use reference temperature and salinity
-                                ! ts_range - use range of temperature and salinity
-                                ! (T_REF and S_REF) to determine surface density
-                                ! and GINT calculate internal densities.
-                                ! gprime - use reference density (RHO_0) for surface
-                                ! density and GINT calculate internal densities.
-                                ! ts_profile - use temperature and salinity profiles
-                                ! (read from COORD_FILE) to set layer densities.
-                                ! USER - call a user modified routine.
-                                ! The file from which the coordinate densities are read."
+NJHALO = 4
+                                ! "default = 2
+                                ! The number of halo points on each side in the
+                                ! y-direction.  With STATIC_MEMORY_ this is set as NJHALO_
+                                ! in MOM_memory.h at compile time; without STATIC_MEMORY_
+                                ! the default is NJHALO_ in MOM_memory.h (if defined) or 2."
 
-REMAP_UV_USING_OLD_ALG = False
-                                ! "[Boolean] default = True
-                                ! If true, uses the old remapping-via-a-delta-z method for remapping u and v. If
-                                ! false, uses the new method that remaps between grids described by an old and
-                                ! new thickness."
+NIGLOBAL = 360
+                                ! "The total number of thickness grid points in the
+                                ! x-direction in the physical domain. With STATIC_MEMORY_
+                                ! this is set in MOM_memory.h at compile time."
+
+NJGLOBAL = 300
+                                ! "The total number of thickness grid points in the
+                                ! y-direction in the physical domain. With STATIC_MEMORY_
+                                ! this is set in MOM_memory.h at compile time.
+                                ! The number of processors in the x-direction. With
+                                ! STATIC_MEMORY_ this is set in MOM_memory.h at compile time.
+                                ! The number of processors in the x-direction. With
+                                ! STATIC_MEMORY_ this is set in MOM_memory.h at compile time.
+                                ! The processor layout that was acutally used."
+
+IO_LAYOUT = 1, 1
```

Comment:
we'd want to adjust this for a production run, e.g. https://github.com/COSIMA/1deg_jra55_ryf/blob/e9796f2c1617cbc47ba0e1ab25c30fc084779aff/ocean/input.nml#L230-L231

Reply:
Could we think about this in a separate PR? I've opened an issue here: https://github.com/COSIMA/access-om3/issues/54